### PR TITLE
Added reference select usage example.

### PR DESCRIPTION
**Pull request by estambakio-sc**
Added [react-reference-select](https://github.com/OpusCapita/react-reference-select/) field to example model (parent Id search field).  #19

### CODE REVIEW

File: src/demo/models/contracts/index.js [hunk 1]
```diff
@@ -268,10 +269,8 @@ const buildFormLayout = viewName => ({ tab, section, field }) => instance => [
   tab({ name: 'general', columns: 2 }, // Best look with N = 2, 3, 4 (default is 1)
     field({ name: 'contractId', readOnly: viewName !== VIEW_CREATE }),
     field({ name: 'description' }),
-    // field({ name: 'translations', render: { Component: TranslatableTextEditor }}),
     field({ name: 'statusId', render: { Component: StatusField, valueProp: { type: 'number' } } }),
-    // field({ name: 'parentContract', render: { Component: ContractReferenceSearch }}),
-    // field({ name: 'currencyId', render: { Component: CurrencyField }}),
+    field({ name: 'parentContract', render: { Component: ReferenceSearch, valueProp: { type: 'string' } } }),
```
Comment (amourzenkov-sc) [inline src/demo/models/contracts/index.js]: remove valueProp because it has default values

File: src/demo/models/contracts/index.js [hunk 2]
```diff
@@ -248,7 +249,7 @@ export const fields = {
     }
   },
   'parentContract': {
```
Comment (amourzenkov-sc) [inline src/demo/models/contracts/index.js]: make parentContract: {} because it has default values

Comment (estambakio-sc) [reply src/demo/models/contracts/index.js]: Done

File: src/demo/models/contracts/index.js (hunk 1 above)
Comment (estambakio-sc) [reply src/demo/models/contracts/index.js]: Done

**Pull request merged**